spin/pom.xml

```diff
@@ -130,6 +131,15 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>${plugin.version.javadoc}</version>
```

Comment:
plugin versions are managed by root /pom.xml. Remove this line.

Reply:
changed that